Mathlib/Computability/NFA.lean

```diff
@@ -152,6 +251,9 @@ def accepts : Language α := {x | ∃ S ∈ M.accept, S ∈ M.eval x}
 theorem mem_accepts {x : List α} : x ∈ M.accepts ↔ ∃ S ∈ M.accept, S ∈ M.evalFrom M.start x := by
   rfl
 
+theorem accepts_acceptsFrom : M.accepts = M.acceptsFrom M.start := by
```

Comment:
Turn it around and make it simp?

Reply:
I'm not sure? For #30872 and #31610, I use `acceptsFrom` for the main induction proofs, and for the main `M.accepts` result I convert it to `M.acceptsFrom M.start`, and then use the `acceptsFrom` lemma with a general set instead of `M.start`.